docs/principles/version.md

```diff
@@ -139,14 +139,6 @@ GET /path/to/resource?version=2021-09-21
 
 would match only resource versions released as GA, as of 2021-09-21.
 
-As a shortcut, you can also request "today's latest experimental" with:
-
-```json
-GET /path/to/resource?version=experimental
```

Comment:
My little testing on versioning looks to me this is not (no longer?) supported. Is it a legacy behaviour that predates date resolution?

Reply:
It turns out that the node implementation allows this, but it probably shouldn't.

It's useful to allow OpenAPI spec lookup without the date, e.g. `/openapi/experimental`. In that case, you're asking for the latest version of the OpenAPI spec at experimental stability. You might want to see that, just to get a snapshot of what the latest API looks like. The spec will have annotations telling you at which version each endpoint was introduced -- a nice way to discover the API.

Allowing the date to "float" in other API endpoint requests puts more risk onto the API consumer. If an API consumer builds their software against `?version=experimental`, that can easily break. By not allowing that to happen in the first place, we're removing that risk and promoting responsible and correct use of the versioning system from the start.